share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -219,7 +219,7 @@ enrg,energycoin,energy coin,
 eqx,equinoxcoin,equinox coin,
 erm,ermiscoin,ermis coin,
 etc,eternalcoin,eternal coin,
-eth,etherium,etherium coin,
+eth,ether,etherium,etherium coin,ethereum,
```

Comment:
People searching for `10 ethereum to bitcoin` - even though the naming is incorrect - would expect to see a conversion for ether, so I'd keep it in to cover such queries.